src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -242,7 +251,7 @@ public function getNode($path)
 
         if (false !== ($result = $this->caches['nodes']->fetch($cacheKey))) {
             if ('ItemNotFoundException' === $result) {
-                throw new ItemNotFoundException(sprintf('Item "%s" not found in workspace "%s"', $path, $this->workspaceName));
+                throw new ItemNotFoundException("Item '$path' not found in workspace '$this->workspaceName'");
```

Comment:
i prefer sprintf over string inlining. can you please revert this change?

and i thought `$this->workspaceName` would need to be in {} to work.